project/package/CodeEditor.vue

```diff
@@ -386,6 +390,21 @@ export default {
   margin: 0;
 }
 
+count-lines{
+  counter-reset: line;
+}
```

Comment:
Change line 396 to .count-lines:deep(span){
So that every span gets a line number, regardless of the class highlight js gives.

Reply:
in this line I miss the dot before the class, anyway what you say isn't possible because the :deep(span) would reset all counters this mean will appear 1 before all lines

Look the screenshot

![Schermata 2022-10-18 alle 11 34 00](https://user-images.githubusercontent.com/36926081/196394252-6fee0870-011d-487c-aa43-3889214dfc9e.png)